api/types/access_request.go

```diff
@@ -46,11 +46,11 @@ type AccessRequest interface {
 	GetCreationTime() time.Time
 	// SetCreationTime sets the creation time of the request.
 	SetCreationTime(time.Time)
-	// GetAccessExpiry gets the upper limit for which this request
-	// may be considered active.
+	// GetAccessExpiry gets the TTL for the elevated certificate that will be
+	// issued if the Access Request is approved.
 	GetAccessExpiry() time.Time
```

Comment:
I think the godoc here is incorrect. If it returned the _TTL_ I would expect the return type to be a `time.Duration`, not a `time.Time`. This returns the expiration time, not the TTL, right?